api/users/schemas.py

```diff
@@ -0,0 +1,8 @@
+import os
+import json
+
+here = os.path.split(os.path.abspath(__file__))[0]
+
+def from_json(fname):
```

Comment:
This file was moved to a directory. I'm getting an `ImportError` when initializing the app. I think this file just needs to be removed.